### PR TITLE
Connector limitation documentation

### PR DESCRIPTION
**Pull request by guiangumpac**
### Description
Added limitations and issues table to PowerBI connector markdown file for users' and developers' knowledge.
 
### Check List
- [ ] New functionality includes testing.
  - [ ] All tests pass, including unit test, integration test and doctest
- [ ] New functionality has been documented.
  - [ ] New functionality has javadoc added
  - [ ] New functionality has user manual doc added
- [ ] Commits are signed per the DCO using --signoff 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: bi-connectors/PowerBIConnector/docs/AmazonOpenSearchService.md [hunk 1]
```diff
@@ -64,3 +64,16 @@
...
+| [Visualizations with aggregate functions selected are not yet supported](https://github.com/opensearch-project/sql/issues/363) | Specifically Minimum, Maximum, Standard deviation, Variance, or Median options are not yet supported. |
+| [Basic filtering limitations](https://github.com/opensearch-project/sql/issues/347) | Selecting more than one value is not yet supported when using basic filtering. Due to the issue in the first row of this table, selecting one value for basic filtering when there is no numerical column is not yet supported. |
+| [Top N filtering limitations](https://opensearch.org/docs/latest/search-plugins/sql/limitation/) | OpenSearch has limitations on subquery which does not yet support the Top N filtering functionality. |
+| [Advanced filtering limitations](https://github.com/opensearch-project/sql/issues/308) | `does not contain` and `does not start with` filters for string columns are not yet supported. All advanced filtering for numerical columns are not yet supported except for `is` and `is blank`. All advanced filtering for date and time columns are not yet supported except for `is blank` and `is not blank`. |
```
Comment (joshuali925) [inline bi-connectors/PowerBIConnector/docs/AmazonOpenSearchService.md:78]: curious what is advanced filtering? the linked issue is regarding new engine not supporting `IN` keyword

**Review by joshuali925 — APPROVED** [review]

File: bi-connectors/PowerBIConnector/docs/AmazonOpenSearchService.md (hunk 1 above)
Comment (guiangumpac) [reply bi-connectors/PowerBIConnector/docs/AmazonOpenSearchService.md:78]: Advanced filtering is a filtering option in PowerBI. The sql query advanced filtering generates uses the `IN` keyword which is an issue tracked through the link included.

Comment (joshuali925) [reply bi-connectors/PowerBIConnector/docs/AmazonOpenSearchService.md:78]: Got it, thanks

**Review by ps48 — APPROVED** [review]
Thanks 👍🏽

**Pull request merged**